sdk/storage/Azure.Storage.Queues/tests/ClientSideEncryptionTests.cs

```diff
@@ -188,7 +192,11 @@ private Mock<Microsoft.Azure.KeyVault.Core.IKey> GetTrackOneIKey(byte[] userKeyB
             {
                 const int keySizeBits = 256;
                 var bytes = new byte[keySizeBits >> 3];
+#if NET6_0_OR_GREATER
+                RandomNumberGenerator.Create().GetBytes(bytes);
+#else
                 new RNGCryptoServiceProvider().GetBytes(bytes);
```

Comment:
Shoud we add a TestRandomNumberGenerator ?

Reply:
What scenario did you have in mind that wouldn't be covered by `TestRecording.Random`?

Reply:
The one we are adding `#if NET6_0_OR_GREATER` for. There are similar usages in the test framework itself.

Reply:
Maybe we should extend TestRecording.Random to do RGN byte generation